src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+
+  if (days < 3) {
+    return (days * rentalCost);
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days < 7) {
```
Цю перевірку можна прибрати. Якщо перша перевірка не пройшла, то днів точно буде >=3

Reply:
Done!